src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,129 @@
+import React, {PropTypes} from 'react';
+
+import Mixins from '../mixins';
+import {getMuiTheme} from '../styles';
+
+const Stepper = React.createClass({
+  propTypes: {
+    /**
+     * The current active step index which passed by parent component
+     */
+    activeStepIndex: PropTypes.number,
+    /**
+     * Children should be Step type
+     */
+    children: PropTypes.node,
+    /**
+     * The icon will show when the step is completed
+     */
+    completedIcon: PropTypes.node,
```

Comment:
Let's see what @oliviertassinari @alitaheri @mbrookes thinks...

I suppose another possible benefit of that approach is it would be easier to change icons for validation. Like this:

![stepper-icon-validation](https://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0B7WCemMG6e0VMWxJVzRSTGFfTGc/components_acux_stepper_altlblplerr_rl.png)